### ISSUE TITLE
Uncaught TypeError: SvgPanZoomRotate is not a function

### ISSUE THREAD

**Issue by BitPhinix** [bastienmoulia/svg-pan-zoom-rotate#1]
Hi, I think your ts typings aren´t correct. When I want to create a new instance I get the Exception "Uncaught TypeError: SvgPanZoomRotate is not a function". 

Here is my code :

```
import SvgPanZoomRotate = require("svg-pan-zoom-rotate");

export class Map2d {
    private svgElement: SvgPanZoomRotate.Instance;
    private htmlElement: HTMLElement;

    constructor(element: HTMLElement, path: string) {
        this.htmlElement = element;
        
        this.svgElement = SvgPanZoomRotate(element, {
            panEnabled: true,
            controlIconsEnabled: false,
            zoomEnabled: true,
            dblClickZoomEnabled: true,
            mouseWheelZoomEnabled: true,
            preventMouseEventsDefault: true,
            zoomScaleSensitivity: 0.2,
            minZoom: 0.5,
            maxZoom: 10,
            fit: true,
            contain: false,
            center: true,
            refreshRate: 'auto'
        });

        this.load(path);
    }

    load(path: string) {
        this.htmlElement.setAttribute("data", path);
    }
}
```

Hope you can help,

Bitphinix

**Comment by bradleyDean** [comment]
I had the same problem. 

Here's how I solved it:
1.) use the init method
svgPanZoom.init(--your svg--);

2.) update the interface in the svg-pan-zoom.d.ts file: 
under Interface Instance, on line 46, add this line:

init: (options: CustomEventOptions) => void;

...I put it on line 46. Now I can initialize a panZoomRotate instance!

**Comment by bastienmoulia** [comment]
@bradleyDean Thank you for the tip!
@BitPhinix Have you try to import with `import * as SvgPanZoomRotate from "svg-pan-zoom-rotate";` ?

**Comment by bradleyDean** [comment]
that is how I imported, yes

One thing to note about the library: I noticed that the library does not supply a way to rotate the svg about a particular point and it seems to not provide any useful information for converting tap coordinates into the reference frame of the rotated svg. I have used the non-rotating version of svgPanZoom and it was pretty easy to convert tap coordinates, but this library is doing some weird stuff to the pan object when it rotates. If you manually pan to 0,0 then rotate (for example) the image doesn't move (to 0,0) ...it just seems to redefine the new position as 0,0. 
For these reasons, I'm planning on just building on the original version of the library to implement my own rotation methods.
